presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -383,6 +383,88 @@ or credentials for a specific use case (e.g., bucket/user specific credentials).
 This Hadoop configuration property must be set in the Hadoop configuration
 files referenced by the ``hive.config.resources`` Hive connector property.
 
+AWS Security Mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presto supports flexible security mapping for AWS Lake Formation(and AWS Glue) and AWS S3 API calls, allowing for separate
+credentials or IAM roles for specific users.
+
+A security mapping entry can be of two types: ``s3`` or ``lakeformation``
+
+Each s3 security mapping entry may specify one match criteria. Available match criteria:
+
+* ``user``: Regular expression to match against username. Example: ``alice|bob``
+
+The security mapping must provide one or more configuration settings:
+
+* ``accessKey`` and ``secretKey``: AWS access key and secret key. This overrides
+  any globally configured credentials, such as access key or instance credentials.
+
+* ``iamRole``: IAM role to use. This overrides any globally configured IAM role.
+
+Each lakeformation security mapping entry may specify one match criteria. Available match criteria:
+
+* ``user``: Regular expression to match against username. Example: ``alice|bob``
+
+The security mapping must provide one configuration setting:
+
+* ``iamRole``: IAM role to use. This overrides any globally configured IAM role.
+
+The security mapping entries are processed in the order listed in the configuration
```

Comment:
If you move the s3 example file as I suggest, also move this paragraph after line 392 (A security mapping entry can be of two types: s3 or lakeformation.), because that sentence and this paragraph describe all security mapping entries, both s3 and lakeformation. 

Moving the s3 example leaves this paragraph in the lakeformation section so this should move to where it applies to all security mapping entries.

Reply:
Makes sense, I have re-ordered as you suggested.